### ISSUE TITLE
[Bug] Window stays open

### ISSUE THREAD

**Issue by justinstanford** [gileli121/WindowTop#4]
After upgrading to v3.3.4 I am unable to close a blank windowtop.exe window.  This window remains open at all times when Windowtop.exe is running.  The only way to close the window is to terminate the windowtop.exe process, however that action kills all Windowtop functionality.  When relaunching again, even after reinstall/reboot the same behavior is apparent.

I am on the latest Windows 10 version 1903, OS build 18362.175 - fully updated as of 7/6/19.  Screenshot of window here:  [Image](https://ibb.co/wSbpdpR)

**Comment by gileli121** [comment]
Hello, 
Thanks for the report.

1. What version did you use before?
2. When you used the old version, you used the same windows 10 version?
3. Did I understand correctly that the bug does not have any specific steps? You only launching WindowTop and the bug occurs? 

Thanks for the report.
This is a high priority issue

**Comment by gileli121** [comment]
Hello, I could not reproduce the bug even after windows update 1903.
Please describe what exactly you did to get the issue.

Thanks.

**Comment by justinstanford** [comment]
Hello and thank you for your responses.  I just upgraded to v3.3.5 today 7/10/19 and this issue seems to have been resolved.  Previously I was on v3.2.4 and saw the issue on upgrade to v3.3.4- yes on the same version of Windows.  Since it does not appear to be happening to me with the latest version it is safe to close this as solved.  Thank you very much for the wonderful program- happily purchased the PRO license.
